src/test/java/com/commercetools/sync/commons/utils/ResourceIdentifierUtilsTest.java

```diff
@@ -45,6 +52,81 @@ void toResourceIdentifierIfNotNull_WithNonNullReference_ShouldReturnCorrectResou
         assertThat(categoryResourceIdentifier.getTypeId()).isEqualTo(Category.resourceTypeId());
     }
 
+    @Test
+    void isReferenceOfType_WithEmptyObjectNodeValueAndEmptyString_ShouldReturnFalse() {
+        // preparation
+        final ObjectNode emptyObjectNode = JsonNodeFactory.instance.objectNode();
+
+        // test
+        final boolean result = isReferenceOfType(emptyObjectNode, "");
+
+        // assertion
+        assertThat(result).isFalse();
+    }
+
+    @Test
+    void isReferenceOfType_WithEmptyObjectNodeValueIsProductReference_ShouldReturnFalse() {
```

Comment:
I didn't like the `_Product` because it breaks the naming convention for tests.

What do you think about this: https://github.com/commercetools/commercetools-sync-java/pull/444/commits/0d943cba1530f28fcf35c8f722d5e265bfacbb8a?

Reply:
using versus `vs` is a nice idea 👍